defaults/osdefs/osdefs_darwin.go

```diff
@@ -0,0 +1,10 @@
+// +build darwin
+
+package osdefs
+
+const (
+	// PrCapbsetDrop is prctl PR_CAPBSET_READ argument value
+	PrCapbsetDrop = 0x18
```

Comment:
It is. After some discussions with @vdemeester and @vieux, there seems to be no way to access cleanly the linux-specific constants in the `syscall` package and build for OSX so I made a small "shim".